libs/coin-modules/coin-hedera/src/config.ts

```diff
@@ -0,0 +1,20 @@
+import { ConfigInfo } from "@ledgerhq/live-config/LiveConfig";
```

Comment:
You can see other coin-module, but the type of the config of a coin-module, must not depend on LiveConfig package.

Reply:
@sprohaszka-ledger I follow what the guys did for `cosmos` and we do have the same import here https://github.com/LedgerHQ/ledger-live/blob/77846a0be4c2a08dfef0e938e1af1479ad3787b7/libs/coin-modules/coin-cosmos/src/config.ts#L3

I will do the modification on `hedera` but should we open a ticket for `cosmos` ?

Reply:
I'll modify `cosmos` and open up a PR